release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -78,8 +78,159 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
+
+* HyperPAV enabled on IBM Z and LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* The following {product-title} Technology Preview features are unsupported:
+** Precision Time Protocol (PTP) hardware
+
+* The following {product-title} features are unsupported:
+** Automatic repair of damaged machines with machine health checking
+** CodeReady Containers (CRC)
+** Controlling overcommit and managing container density on nodes
+** CSI volume cloning
+** CSI volume snapshots
+** FIPS cryptography
+** NVMe
+** OpenShift Metering
+** OpenShift Virtualization
+** Tang mode disk encryption during {product-title} deployment
+
+* Worker nodes must run {op-system-first}
+* Persistent shared storage must be provisioned by using either NFS or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA
+
 [id="ocp-4-10-ibm-power"]
-=== IBM Power Systems
+=== IBM Power
+
+With this release, IBM Power is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on IBM Power with {product-title} {product-version}:
+
+* Horizontal pod autoscaling
+* The following Multus CNI plug-ins are supported: 
+** Bridge
+** Host-device
+** IPAM
+** IPVLAN
+
+* NMState Operator
+* OVN-Kubernetes IPsec encryption
+* Vertical Pod Autoscaler Operator
+
+[discrete]
+==== Supported features
+
+The following features are also supported on IBM Power:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Local Storage Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** SR-IOV Network Operator
+** Service Binding Operator
+** Vertical Pod Autoscaler Operator
+
+* Encrypting data stored in etcd
+* Helm
+* Multipathing
+* Multus SR-IOV
+* NVMe
+* OVN-Kubernetes
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* Support for multiple network interfaces
+* Support for Power10
+* Three-node cluster support
+* 4K Disk Support
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power:
```

Comment:
Same here. Could we reword this to omit "note"? Perhaps:
The following restrictions impact {product-title} on IBM Power: